http/watcher.go

```diff
@@ -74,11 +73,11 @@ func (w *BTCAddressWatcher) performCallForUser() error {
 		From:     q.LiquidityProviderRskAddress,
 		Signer:   w.lp.SignTx,
 	}
-
 	_, err = w.rsk.CallForUser(opt, q)
 	if err != nil {
 		return err
 	}
+	w.calledForUser = true
```

Comment:
Here we could use GetTxStatus to wait for the tx being mined.

Reply:
I did in the same way the other usages do. Please lemme know if this what you meant.

Reply:
Exported getTxStatus and checking the status in watcher.